cluster-autoscaler/cloudprovider/gce/gce_manager.go

```diff
@@ -101,6 +103,12 @@ type migInformation struct {
 	basename string
 }
 
+type machineTypeKey struct {
+	projectId   string
```

Comment:
Do we need projectId? We don't support running CA across projects anyway.

Reply:
done